### ISSUE TITLE
Support docutils 0.18.x

### ISSUE THREAD

**Issue by befeleme** [readthedocs/sphinx_rtd_theme#1302]
docutils 0.18.x has been with us for quite some time and it has finally started gaining support of major documentation projects, including Sphinx 5.x. Could you also start supporting that version?

**Comment by befeleme** [comment]
In Fedora, I manually relaxed the docutils version requirement and built the package as RPM against the version 0.18.1 & Sphinx 5.0.2.
Please note, we patch our package (especially fonts), so my findings may not be a complete list.
I compared the rendered package documentation to the one build on docutils 0.17 and Sphinx 4.5.0 and found those issues:

1) Field list is rendered with two colons instead of one:
![image](https://user-images.githubusercontent.com/33810531/179508910-1abdf4ab-2f33-42fd-9fc4-fec1d27b7b65.png)

2) Footnotes and citations not visually rendered, functionality (references) without changes:
![image](https://user-images.githubusercontent.com/33810531/179509123-a7c9ccf6-605e-4b57-9d54-b8bd177d2c86.png)

**Comment by n-peugnet** [comment]
@befeleme :
> 1. Field list is rendered with two colons instead of one:

This issue has been [fixed in Sphinx](https://github.com/sphinx-doc/sphinx/issues/10594) and will be [released in version 5.0.3](https://www.sphinx-doc.org/en/master/changes.html#release-5-0-3-in-development)

**Comment by JasperCraeghs** [comment]
> This issue has been https://github.com/sphinx-doc/sphinx/issues/10594 and will be [released in version 5.0.3](https://www.sphinx-doc.org/en/master/changes.html#release-5-0-3-in-development)

The change to Sphinx' style sheet has been released in Sphinx 5.1.0 already. However, sphinx-rtd-theme' style sheet also needs a change like this. Two colons are still rendered with sphinx-rtd-theme and Sphinx 5.1.0.

**Comment by nedbat** [comment]
I'm trying to update the dev dependencies for coverage.py, and get a conflict on docutils, partly because of the requirement in sphinx_rtd_theme:
```
Could not find a version that matches docutils<0.18,<0.20,<0.21,>=0.14,>=0.19,~=0.18.0 (from doc8==1.0.0->-r doc/requirements.in (line 10))
Tried: 0.3, 0.3.5, 0.3.7, 0.3.9, 0.4, 0.5, 0.6, 0.7, 0.8, 0.8.1, 0.9, 0.9.1, 0.10, 0.11, 0.12, 0.12, 0.13.1, 0.13.1, 0.14, 0.14, 0.15, 0.15, 0.15.1.post1, 0.15.2, 0.15.2, 0.16, 0.16, 0.17, 0.17, 0.17.1, 0.17.1, 0.18, 0.18, 0.18.1, 0.18.1, 0.19, 0.19
Skipped pre-versions: 0.14rc1, 0.14rc2, 0.16b0.dev0, 0.16b0.dev0, 0.16rc1, 0.16rc1, 0.17b1, 0.17b1, 0.17.1b1.dev0, 0.17.1b1.dev0, 0.18b1, 0.18b1, 0.18.1b0, 0.18.1b0, 0.19b1, 0.19b1
There are incompatible versions in the resolved dependencies:
  docutils<0.18 (from sphinx_rtd_theme==1.0.0->-r doc/requirements.in (line 14))
  docutils<0.21,>=0.19 (from doc8==1.0.0->-r doc/requirements.in (line 10))
  docutils~=0.18.0 (from sphinx-tabs==3.4.1->-r doc/requirements.in (line 15))
  docutils<0.20,>=0.14 (from sphinx==5.1.1->-r doc/requirements.in (line 12))
```
Can we loosen this requirement?
